gobgp/cmd/mrt.go

```diff
@@ -45,7 +44,7 @@ func injectMrt() error {
 		return fmt.Errorf("Specified queue size is smaller than 1, refusing to run with unbounded memory usage")
```

Comment:
For errors like these we can make variables at the top of the file to make these more readable and testable.

```
var (
    ...
    errQueueSizeTooSmall = errors.New("refusing to run with unbounded memory usage, specified queue size is smaller than 1")
)
```

Reply:
Thanks, however it will leads a lot of fixes over the whole code.
I think it is better to do it in another PR.